sandbox/sandbox.py

```diff
@@ -0,0 +1,18 @@
+import os
+import numpy as np
+import pandas as pd
+import tensorflow as tf
+from logging import getLogger, config
+
+logger = getLogger(__name__)
+
+
+def main():
+    x = np.array([0, 1, 2])
+    y = np.array([3, 4])
+    z = np.concatenate([x, y])
+    print(z)
+
+
+if __name__ == '__main__':
+    main()
```

Comment:
[ASK] Is this needed?

Reply:
No, this is not needed. I delete this file.